sample-relayer-config.json

```diff
@@ -9,10 +10,10 @@
       "rpc-endpoint": "https://api.avax-test.network/ext/bc/C/rpc",
       "ws-endpoint": "wss://api.avax-test.network/ext/bc/C/ws",
       "message-contracts": {
-        "0x253b2784c75e510dD0fF1da844684a1aC0aa5fcf": {
+        "0xF7cBd95f1355f0d8d659864b92e2e9fbfaB786f7": {
```

Comment:
Did you mean to include the changes in this file?

Reply:
Oops, no I did not. Reverted.